web/src/features/App/api/useLicense.ts

```diff
@@ -7,24 +7,20 @@ function useLicenseWithIntercept() {
 
   let timerId = useRef<null | NodeJS.Timeout>(null);
 
-  // DEV: sleep function to delay the return of the data
-  // use this to simulate a slow loading request
-  // const sleep = (ms = 1000): Promise<void> => {
-  //   return new Promise<void>((resolve) => {
-  //     setTimeout(() => resolve(), ms);
-  //   });
-  // };
-
   let licenseQuery = useLicense();
 
   useEffect(() => {
     axios.interceptors.request.use(
       (x) => {
-        // not sure if this check is enough to make sure that it
-        // only happens on /license endpoint
         if (x.url?.endsWith("/license")) {
+          if (timerId.current) {
+            clearTimeout(timerId.current);
+            timerId.current = null;
+          }
           // set timeout to 500ms, change it to whatever you want
-          timerId.current = setTimeout(() => setIsSlowLoading(true), 500);
+          timerId.current = setTimeout(() => {
+            setIsSlowLoading(true);
+          }, 1000);
```

Comment:
will refactor these interceptors so that they're more consolidated in a future PR. part of that will include turning this into a constant that can be set globally. We don't really want to have multiple thresholds since that can cause janky loaders flickering in an out.